.github/workflows/build.yaml

```
@@ -0,0 +1,223 @@
+
+env:
+  DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        service: [ "${{ github.event.inputs.service }}" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)